### PR TITLE
1.1 Fixed Repo for Merge

### LINKED ISSUE #534: Compile errors with 1.1 KSP .dll-s
The source needs modifications to be able to compile with v1.1 KSP .dll-s. Guidelines to fix some of the issues: http://forum.kerbalspaceprogram.com/index.php?/topic/135179-info-on-how-to-convert-your-plugin-to-ksp-11/

### LINKED ISSUE #535: Replace RenderingManager related source code
RenderingManager shouldn't be used in 1.1 KSP. More information: http://forum.kerbalspaceprogram.com/index.php?/topic/135179-info-on-how-to-convert-your-plugin-to-ksp-11/

### LINKED ISSUE #536: Update Staging related source code
Replace `Staging.ActivateNextStage();` with `StageManager.ActivateNextStage();`

http://forum.kerbalspaceprogram.com/index.php?/topic/75245-105-remotetech-v169-2015-11-10/&do=findComment&comment=2495421

### LINKED ISSUE #537: "Delay time" GUI element is missing
"Delay time" GUI element along with flight computer icon is missing in flight and map mode.

Please refer to these posts for more information and possible fixes: http://forum.kerbalspaceprogram.com/index.php?/topic/75245-105-remotetech-v169-2015-11-10/&page=168

@Koritastic has made some fixes regarding this issue (I haven't verified): https://github.com/Koritastic/RemoteTech/commits/develop/src/RemoteTech
especially these commits:
https://github.com/Koritastic/RemoteTech/commit/6da42b8ebc7f15a644e230fc56311283393128a0
https://github.com/Koritastic/RemoteTech/commit/575965b56f0a4cca02582e0eb8a5028d588f451d

### LINKED ISSUE #538: Filter Overlay GUI element needs to be repositioned
The Filter Overlay GUI element (where you can set which connection lines are visible) that shows up in tracking station and map mode needs repositioning. Currently it displays over other GUI elements. Filter Overlay source code can be found in FilterOverlay.cs.

### LINKED ISSUE #539: Focus Overlay GUI element needs repositioning.
Focus Overlay GUI element is a button that displays list of vessels that can be focused, it is available in map and tracking station scene.
In KSP 1.1, the Knowldege Base panel has been overhauled. The Focus Overlay needs to be integrated to the new Knowledge Base panel. The source code of the Focus Overlay GUI element can be found in FocusOverlay.cs.

### LINKED ISSUE #542: Some textures don't render in 1.1
Flight computer and a few other textures don't display in the new 1.1 build.

### LINKED ISSUE #543: Using 'KILL' or 'NODE' causes the craft to spin
See http://forum.kerbalspaceprogram.com/index.php?/topic/75245-105-remotetech-v169-2015-11-10/&page=169

### LINKED ISSUE #546: Update ModuleManager to latest version.
Update ModuleManager version that is included in the release bundle.

### PR DESCRIPTION
**Pull request by Koritastic**
Fixes #538, fixes #539, fixes #537, fixes #535, fixes #534, fixes #543, fixes #543, fixes #546

Fixes #536 - Look into the "bug" this creates, See [THIS ](http://forum.kerbalspaceprogram.com/index.php?/topic/136073-edit-not-quite-true-in-11-autopilot-mods-denied-the-ability-to-stage-during-map-view/)forum post for details.

Fixes #542 by replacing manual texture loading with Unity texture loading & moved textures into their own directory.

### CODE REVIEW

File: src/RemoteTech/FlightComputer/Commands/ActionGroupCommand.cs [hunk 1]
```diff
@@ -17,9 +17,9 @@ public override bool Pop(FlightComputer f)
             f.Vessel.ActionGroups.ToggleGroup(ActionGroup);
             if (ActionGroup == KSPActionGroup.Stage && !(f.Vessel == FlightGlobals.ActiveVessel && FlightInputHandler.fetch.stageLock))
             {
-                Staging.ActivateNextStage();
-                ResourceDisplay.Instance.Refresh();
-            }
+				KSP.UI.Screens.StageManager.ActivateNextStage();
+				KSP.UI.Screens.ResourceDisplay.Instance.Refresh();
+			}
```
Comment (tomekpiotrowski) [inline src/RemoteTech/FlightComputer/Commands/ActionGroupCommand.cs]: I know we've talked about it on Slack, but just to document it: indentation seems to be off in some parts of this PR. This is one example.

File: src/RemoteTech/RemoteTech.csproj [hunk 2]
```diff
@@ -16,26 +16,46 @@
...
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
   </PropertyGroup>
+  <PropertyGroup>
+    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
+  </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>Assembly-CSharp.dll</HintPath>
-      <Private>False</Private>
+      <HintPath>H:\Steam\steamapps\common\Kerbal Space Program1.1\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: We can't have those hard-coded paths in the project configuration. Everything should be relative to the repository. You're free to change locations of dependencies in your local repository, but pushing them to the main repo would change them for all developers.

File: src/RemoteTech/RemoteTech.csproj [hunk 3]
```diff
@@ -131,46 +155,18 @@
...
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Resources\texBackground_left.png" />
-  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
+  <PropertyGroup>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: Same comment as above applies. That's actually a neat piece of configuration, I might steal it myself, but we shouldn't upload it to the main repo.

**Comment by tomekpiotrowski** [comment]
Why was `GameData/ModuleManager.2.6.13.dll` removed?

File: .gitignore [hunk 4]
```diff
@@ -158,6 +158,5 @@ $RECYCLE.BIN/
 # Ignore dependencies and builds
 src/RemoteTech/*.dll
 *.zip
-GameData/RemoteTech/Plugins/RemoteTech.dll
```
Comment (tomekpiotrowski) [inline .gitignore]: I think we should keep that line. We don't want .dll files in the repo.

**Comment by d4rksh4de** [comment]
Fixes #543

File: src/RemoteTech/Modules/ModuleRTDataTransmitter.cs [hunk 5]
```diff
@@ -122,6 +122,7 @@ private IEnumerator Transmit(Callback callback = null)
                     }
                     yield return new WaitForSeconds(PacketInterval);
                 }
+                GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel);
```
Comment (d4rksh4de) [inline src/RemoteTech/Modules/ModuleRTDataTransmitter.cs]: Doesn't compile for me, it needs 3 arguments: Fire(ScienceData, Vessel, bool)

**Comment by d4rksh4de** [comment]
Fixes #546

File: src/RemoteTech/Modules/ModuleRTDataTransmitter.cs (hunk 5 above)
Comment (d4rksh4de) [inline src/RemoteTech/Modules/ModuleRTDataTransmitter.cs]: Tried both ways, 
`GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel, true);`
and
`GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel, false);`
it doesn't fix MPL bug described #511

File: src/RemoteTech/RemoteTech.csproj [hunk 6]
```diff
@@ -21,7 +21,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <AllowUnsafeBlocks>false</AllowUnsafeBlocks>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: Why do we need this?

File: src/RemoteTech/RemoteTech.csproj [hunk 7]
```diff
@@ -32,10 +32,30 @@
     <WarningLevel>4</WarningLevel>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
   </PropertyGroup>
+  <PropertyGroup>
+    <RunPostBuildEvent>OnBuildSuccess</RunPostBuildEvent>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: This is not needed now that you've removed the PostBuildEvent.

File: src/RemoteTech/RemoteTech.csproj [hunk 8]
```diff
@@ -131,46 +155,13 @@
...
-  <ItemGroup>
-    <EmbeddedResource Include="Resources\texBackground_left.png" />
-  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Target Name="BeforeBuild">
   </Target>
   <Target Name="AfterBuild">
   </Target>
+  <PropertyGroup>
+    <PostBuildEvent>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: It's good that you've removed the event itself. Those lines look like they're not needed anymore.

File: src/RemoteTech/RemoteTech.csproj [hunk 9]
```diff
@@ -16,7 +16,7 @@
     <DebugSymbols>false</DebugSymbols>
     <DebugType>none</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\GameData\RemoteTech\Plugins\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: I'd prefer to leave that path the way it was.

File: src/RemoteTech/RemoteTech.csproj [hunk 10]
```diff
@@ -26,7 +26,7 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
     <DebugType>none</DebugType>
     <Optimize>true</Optimize>
-    <OutputPath>..\..\GameData\RemoteTech\Plugins\</OutputPath>
+    <OutputPath>bin\Release\</OutputPath>
```
Comment (tomekpiotrowski) [inline src/RemoteTech/RemoteTech.csproj]: Same here.

**Pull request merged**